pyrtl/helperfuncs.py

```diff
@@ -150,7 +127,23 @@ def probe(w, name=None):
     confuse various post-processing transforms such as output to verilog)
     """
     global probeIndexer
-    return probeIndexer.next_indexed(w, name)
+    index = probeIndexer.next_index()
+    prefix = probeIndexer.internal_prefix
```

Comment:
`_NameIndexer` only has two methods and one of them is called here. The other doesn't go along with the naming convention for probes